lib/src/interfaces/Options.ts

```diff
@@ -387,6 +386,26 @@ export interface OptionsTopBarBackButton {
    * #### (iOS specific)
    */
   displayMode?: 'default' | 'generic' | 'minimal';
+  /**
+   * Controls whether the default back button should pop the Stack or not
+   * @default true
+   */
+  popStackOnPress?: boolean;
+}
+
+export interface HardwareBackButtonOptions {
+  /**
+   * Controls whether the hardware back button should dismiss modal or not
+   * #### (Android specific)
+   * @default true
+   */
+  dismissModalOnPress?: boolean;
+  /**
+   * Controls whether the hardware back button should pop the Stack or not
+   * #### (Android specific)
```

Comment:
popStackOnPress is relevant for both platforms

Reply:
But not for the hardware button

Reply:
w00ps my bad